azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -321,10 +322,6 @@ func flattenAzureRmKubernetesClusterAgentPoolProfiles(profiles *[]containerservi
 			agentPoolProfile["dns_prefix"] = *profile.DNSPrefix
 		}
 
-		if profile.Fqdn != nil {
```

Comment:
Can just be set from profile.Fqdn for now.

Reply:
I'd be surprised that it would break existing code (`profile.Fqdn` has always been empty, so the field hasn't been useful), but I can make that change and deprecate it.